src/utils/moneyAmount.ts

```diff
@@ -0,0 +1,16 @@
+import BigNumber from 'bignumber.js'
+
+export interface LocalMoneyAmount {
+  value: BigNumber.Value
+  currencyCode: string
+  exchangeRate: BigNumber.Value
+}
+
+/**
+ * @deprecated Use TokenAmount instead
```

Comment:
do we maybe want to put these types with TokenAmount then?

Reply:
I'm not too sure yet, TokenAmount is currently within `src/transactions/types`, this type is more generic.